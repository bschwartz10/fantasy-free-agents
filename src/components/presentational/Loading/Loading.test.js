import React from 'react'
import { shallow } from 'enzyme'
import Loading from './Loading'

describe('Loading component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })

  it('should exist', () => {
    expect(wrapper).toBeDefined()
  })

  it('should have div with className of spinner', () => {
    const div = wrapper.find('.spinner')

    expect(div).toBeDefined()
  })

  it('should have div with className of double-bounce1', () => {
    const div = wrapper.find('.double-bounce1')

    expect(div).toBeDefined()
  })

  it('should have div with className of double-bounce2', () => {
    const div = wrapper.find('.double-bounce2')

    expect(div).toBeDefined()
  })

  it('should match snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
