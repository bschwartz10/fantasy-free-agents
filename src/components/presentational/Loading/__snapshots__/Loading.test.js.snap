// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should match snapshot with all data passed in correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="double-bounce1"
  />
  <div
    className="double-bounce2"
  />
</div>
`;
